examples/homepage/install/arm64/terraform.tf

```
@@ -33,9 +33,9 @@ resource "aws_lambda_function" "test_lambda" {
   function_name = "lambda_function_name"
   role          = aws_iam_role.iam_for_lambda.arn
   handler       = "index.test"
-  runtime       = "python3.9"
-  layers        = ["arn:aws:lambda:{region}:017000801446:layer:AWSLambdaPowertoolsPythonV2-Arm64:69"]
+  runtime       = "python3.12"
+  layers        = ["arn:aws:lambda:{region}:017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:69"]
```

```suggestion
  layers        = ["arn:aws:lambda:{region}:017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:1"]
```

question: version 1 like others? I suspect it'll be fixed as part of the release like always but just in case :D

---

I had to update this with the new name format we decided on. It's up to date.